examples/creating_even_spaced_wavelength_visualisation.py

```diff
@@ -0,0 +1,51 @@
+"""
+=================
+Combining a Celestial WCS with a Wavelength Axis and Building a Spectral-Image Cube
+=================
+
+This example shows how to add an arbitrarily spaced wavelength dimension to a
+celestial WCS.  It then combines the new WCS with a stack of AIA images at
+different wavelengths, and produces an interactive figure of the observations.
```

Comment:
```suggestion
The goal of this example is to construct a spectral-image cube of AIA images at different wavelength.

This will showcase how to add an arbitrarily spaced wavelength dimension to a
celestial WCS.
```